src/Jobs/ImportGroupsJob.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace JustBetter\MagentoPrices\Jobs;
+
+use Illuminate\Bus\Batchable;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use JustBetter\MagentoPrices\Contracts\ImportsGroups;
+
+class ImportGroupsJob implements ShouldBeUnique, ShouldQueue
+{
+    use Batchable;
```

Comment:
Does this job need to be batchable?

Reply:
No, it does not. The trait was added because most jobs have it, I will remove it.